From 87046956fcf021b39f433d052b94b1ed32fb9701 Mon Sep 17 00:00:00 2001
From: Richard Zhu <r65037@freescale.com>
Date: Wed, 15 Apr 2009 11:38:35 +0800
Subject: [PATCH] ENGR00111242-2-MX51 Enable the CONFIG_MMC_UNSAFE_RESUME option

Enable the CONFIG_MMC_UNSAFE_RESUME option, to support
the mmc/sd rootfs.

Signed-off-by: Richard Zhu <r65037@freescale.com>
---
 arch/arm/configs/imx51_3stack_defconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/configs/imx51_3stack_defconfig b/arch/arm/configs/imx51_3stack_defconfig
index 6853951..f7daeba 100644
--- a/arch/arm/configs/imx51_3stack_defconfig
+++ b/arch/arm/configs/imx51_3stack_defconfig
@@ -1355,7 +1355,7 @@ CONFIG_USB_G_SERIAL=m
 # CONFIG_USB_G_PRINTER is not set
 CONFIG_MMC=y
 # CONFIG_MMC_DEBUG is not set
-# CONFIG_MMC_UNSAFE_RESUME is not set
+CONFIG_MMC_UNSAFE_RESUME=y
 
 #
 # MMC/SD Card Drivers
-- 
1.5.4.4

